### PR TITLE
Design About Page

### PR DESCRIPTION
**Pull request by tanvirchahal**
- Added navigation bar
- Added footer 
- Design About page as per mockup https://invis.io/AVRZ389BW8K

### CODE REVIEW

File: ckanext/ed/templates/header.html [hunk 1]
```diff
@@ -60,14 +59,62 @@
...
-    ('search', _('Datasets')),
-    ('organizations_index', _('Organizations')),
-    ('group_index', _('Groups')),
-    ('ckanext_showcase_index', _('Applications')),
-    ('about', _('About'))
-  ) }}
-{% endblock %}
+  <header class="navbar navbar-static-top masthead" >
+    {% block header_debug %} {% if g.debug and not g.debug_supress_header %}
+    <div class="debug">Controller : {{ c.controller }}<br/>Action : {{ c.action }}</div>
```
Comment (zelima) [inline ckanext/ed/templates/header.html]: Think this line is visible on page

Comment (zelima) [reply ckanext/ed/templates/header.html]: image

Comment (tanvirchahal) [reply ckanext/ed/templates/header.html]: @zelima , i think this comes from CKAN when in debugging mode.  I am certain it'll be automatically removed when loaded outside development mode.

Comment (zelima) [reply ckanext/ed/templates/header.html]: OK, let me try turning development mode off

**Pull request merged**